src/tests/utils/setup.ts

```diff
@@ -79,9 +148,59 @@ beforeAll(async () => {
     query: async (sqlString: string) => {
       return await orm.connection.query(sqlString);
     },
+
+    insertNote: async (note: {creatorId: number, content?: JSON, publicId: string}) => {
```

Comment:
maybe define these functions in separate files and import and use here?

Reply:
done